src/main/webapp/app/shared/connection-warning/connection-warning.component.ts

```diff
@@ -43,7 +43,7 @@ export class JhiConnectionWarningComponent implements OnInit, OnDestroy {
     }
 
     ngOnDestroy() {
-        this.websocketStatusSubscription.unsubscribe();
-        this.routerSubscription.unsubscribe();
+        this.websocketStatusSubscription?.unsubscribe();
```

Comment:
Why is this needed? According to the type, this value should not be undefined. Please change the type if this is now the case.